tests/wpt/mozilla/tests/mozilla/interfaces.https.html

```diff
@@ -239,6 +240,7 @@
   "StorageEvent",
   "StyleSheet",
   "StyleSheetList",
+  "SubtleCrypto",
```

Comment:
We should probably gate these new interfaces behind a pref until more if the functionality is present.

Reply:
Ah good point, I'll add that